### DISCUSSION TITLE
Official Carbon Language

### DISCUSSION THREAD

**Discussion by anton-roos** [carbon-language/carbon-lang#1640] [General]
Will the Carbon documentation, examples and guides be in International English or American English?

**Comment by geoffromer** [comment]
We currently use the [Google developer documentation style guide](https://developers.google.com/style), which recommends using American spelling and capitalization conventions.

**Reply by MichaelBonnet** [reply]
Is documentation in other languages in the (I assume far in the future) cards?

**Reply by anton-roos** [reply]
> We currently use the [Google developer documentation style guide](https://developers.google.com/style), which recommends using American spelling and capitalization conventions.

Thank you.
